packages/driver/cypress/e2e/commands/aliasing.cy.ts

```diff
@@ -120,6 +124,7 @@ describe('src/cy/commands/aliasing', () => {
 
       afterEach(function () {
         if (!this.foo) {
+          // @ts-expect-error TODO: mocha Runnable is not expecting error here
           this.test.error(new Error('this.foo not defined'))
```

Comment:
I'm not even sure it this should be seen as a Mocha Runnable, but I certainly didn't want to touch these types